pubtools/_quay/exceptions.py

```diff
@@ -4,3 +4,19 @@ class ManifestTypeError(Exception):
 
 class RegistryAuthError(Exception):
     """Occurres when registry authentication encounters an issue."""
+
+
+class BadPushItem(Exception):
+    """Occurres when a bad push item is being proccessed."""
```

Comment:
```suggestion
    """Occurs when a bad push item is being proccessed."""
```

(and the two classes following this)

Reply:
Fixed.